### ISSUE TITLE
qapi files not found

### ISSUE THREAD

**Issue by hstriepe** [utmapp/UTM#2399]
*I am trying to compile on an M1 after running both Intel and ARM macOS scripts.
There are two files that do not appear to be generated:
qapi-builtin-types.
qapi-types-error.h

**Comment by osy** [comment]
Please read the readme

**Comment by hstriepe** [comment]
I did:
- clone via git
- added brew: brew install bison pkg-config gettext glib libgpg-error nasm make meson and checked bison path
- git submodule update --init --recursive
Ran both dependencies generators:
-./scripts/build_dependencies.sh -p macos -a arm64
-./scripts/build_dependencies.sh -p macos -a x86_64

Notable log entries::
Program scripts/qapi-gen.py found: YES (/opt/homebrew/opt/python@3.9/bin/python3.9 /Users/Shared/repo/git/utm/build-macOS-arm64/qemu-5.2.0-asi/scripts/qapi-gen.py)
[480/5927] Generating Test QAPI files with a custom command
[481/5927] Generating QAPI test (include) with a custom command
[482/5927] Generating QGA QAPI files with a custom command
[503/5927] Generating QAPI files for qemu-storage-daemon with a custom command
[600/5927] Generating shared QAPI source files with a custom command
[715/5927] Generating trace-qapi.c with a custom command (wrapped by meson to capture output
etc., etc.
I did not notice any errors.
I am quite familiar with Xcode and Open Source build steps. I did not want to spend the time parsing through the scripts.
